backend/workspaces/views.py

```diff
@@ -465,6 +465,11 @@ def assign_manager(self, request, pk=None, *args, **kwargs):
                     {"message": "User with such id does not exist!"},
                     status=status.HTTP_404_NOT_FOUND,
                 )
+            if user.role == User.ANNOTATOR or user.role == User.REVIEWER:
+                return Response(
```

Comment:
Optional suggestion: Since the roles are numerically hierarchical, can we please make it:
```py
if user.role < User.WORKSPACE_MANAGER:
```